gravitee-apim-portal-webui-next/src/app/api-details/api-tab-subscriptions/api-tab-subscriptions.component.html

```diff
@@ -22,6 +22,29 @@
       <p i18n="@@subscribeToApi">Subscribe to our API and your subscription will show up here.</p>
     </div>
   } @else {
-    <p>Coming soon</p>
+    <table mat-table [dataSource]="subscriptions" class="api-tab-subscriptions__table">
+      <ng-container matColumnDef="application">
+        <th mat-header-cell *matHeaderCellDef i18n="@@subscriptionTableColumnApplication" class="m3-title-medium">Application</th>
+        <td mat-cell *matCellDef="let element">{{ retrieveMetadataName(element.application) }}</td>
```

Comment:
Methods should be avoided in the html rendering because they are called incessantly, and in this case, the value will not be changing once rendered. Instead of doing the mapping here, move the mapping to the component after the service returns the Subscription response.

Reply:
Done